class.jetpack.php

```diff
@@ -4098,7 +4098,7 @@ function admin_page_load() {
 		case 'verify_secrets_missing' :
 		case 'verify_secrets_mismatch' :
 			$error = esc_html( $error );
-			$this->error = sprintf( __( '<strong>Your Jetpack has a glitch.</strong>  Something went wrong that&#8217;s never supposed to happen.  Guess you&#8217;re just lucky: %s', 'jetpack' ), "<code>$error</code>" );
+			$this->error = sprintf( __( '<strong>Your Jetpack has a glitch.</strong>  We&#8217;re sorry for the inconvenience, please wait 1-2 hours and try again, if you still get this message, please contact support with this message: %s', 'jetpack' ), "<code>$error</code>" );
```

Comment:
@samhotchkiss perhaps we should add some periods, since there 3 different matters here?
>We're sorry for the inconvenience. Please wait 1-2 hours and try again. If you still get this message, please contact support with this message: